### DISCUSSION TITLE
Small Feature Request

### DISCUSSION THREAD

**Discussion by antmar904** [EricZimmerman/KapeFiles#498] [Ideas]
Hi.

When using Azure Storage, kape checks the permissions to the blob and writes a temporary file "%date%KAPE_SFTP_WRITE_TEST_%guid%". Since the SAS does not have "Delete" permissions that file is left behind.  I use that file to tell when kape started running on a endpoint.  Would it be possible to add the computer name to the name of the file or inside the file so I can easily identify the system?  I know this is a small enhancement but would be a great help.

**Comment by EricZimmerman** [comment]
this should be in next release

**Comment by EricZimmerman** [comment]
this didnt make 1003. i added it as an issue tho for the next one

**Reply by antmar904** [reply]
Bummer ok thanks!

**Comment by AndrewRathbun** [comment]
https://github.com/EricZimmerman/KapeFiles/issues/506

Feature added 👍

**Comment by antmar904** [comment]
Hello all.

Sorry to resurrect an old feature request however when looking at the Azure Storage write test temporary file would it be possible to add the "hostname" to the file name?  I know this is added in the contents of the file but being I run KAPE on a lot of remote computers it would help by just seeing the hostname in the name of the temp write file so I can asure KAPE ran on my targeted endpoints.

Example shown here:

![kape](https://github.com/EricZimmerman/KapeFiles/assets/8560787/19541e86-9085-4fcc-ac2c-b6bc15a3c9e2)

**Reply by AndrewRathbun** [reply]
Made an issue here: https://github.com/EricZimmerman/KapeFiles/issues/875
